### PR TITLE
Sort donation wishes

### PR DESCRIPTION
**Pull request by tongo-angelov**
Allow users to sort donation wishes by date or donation amount

### CODE REVIEW

**Comment by slavcho** [comment]
Can you add some screenshots?

File: src/components/client/campaigns/DonationWishes.tsx [hunk 1]
```diff
@@ -1,30 +1,73 @@
...
+      sortBy: value,
+      sortOrder: sort.sortBy === value ? (sort.sortOrder === 'desc' ? 'asc' : 'desc') : 'desc',
+    }))
+  }
+
+  const handleSearch = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setSearchValue(event.target.value)
+  }
+
+  const debounceSearch = useMemo(() => debounce(handleSearch, 300), [])
```
Comment (slavcho) [inline src/components/client/campaigns/DonationWishes.tsx:69]: nice

**Review by slavcho — COMMENTED** [review]
The code looks good to me.
But I'd like to see some screenshots.
Do we have a ticket for this code?

**Comment by tongo-angelov** [comment]
I'm sorry, here are some screenshots.

Sort by donation amount:
Screenshot 2023-08-08 at 8 31 54

Search:
Screenshot 2023-08-08 at 8 32 18

Search with no results:
Screenshot 2023-08-08 at 8 32 28

**Comment by igoychev** [comment]
> Do we have a ticket for this change?
Yep, https://github.com/podkrepi-bg/frontend/issues/1429

**Comment by slavcho** [comment]
Great, I would only change "Сортирай" with "Сортирай по". 

> I'm sorry, here are some screenshots.
> 
> Sort by donation amount: Screenshot 2023-08-08 at 8 31 54
> 
> Search: Screenshot 2023-08-08 at 8 32 18
> 
> Search with no results: ![Screenshot 2023-08-08 at 8 32 28](https://user-images.githubusercontent.com/123360440/258993904-ebe403fb-12db-4698-951d-890e4dd3262f.

**Review by slavcho — APPROVED** [review]

**Review by ani-kalpachka — APPROVED** [review]

**Comment by ani-kalpachka** [comment]
Is this related to https://github.com/podkrepi-bg/frontend/issues/1399 or https://github.com/podkrepi-bg/frontend/issues/1429? 
I think the two tickets are duplicated.

**Pull request merged**